### PR TITLE
SBT

### PR DESCRIPTION
**Pull request by xssnick**
Implemented SBT smart-contract. It also includes tests and descriptions.

### CODE REVIEW

**Review by Ahmitub — APPROVED** [review]

**Review by TANKA2522 — APPROVED** [review]

**Review by TANKA2522 — APPROVED** [review]

**Review by RodnyPHP — COMMENTED** [review]
Issues Found:

1. SbtItem.data.ts - Lines 153-162: The getStaticData function has repetitive/buggy code - it writes OperationCodes.getStaticData multiple times unnecessarily.

2. sbt.md - Line 89: There’s a typo - }s should be }

Let me fix these issues:I’ve identified and fixed the critical bugs in the code:

Main Issues Found and Fixed:

1. SbtItem.data.ts - `getStaticData` function (Lines 163-173)

Problem: The function writes OperationCodes.getStaticData to the message body 10 times instead of once. This bloats the message and wastes gas.

- [ ] getStaticData: (params: {queryId?: number}) => {
    let msgBody = new Cell()
    msgBody.bits.writeUint(OperationCodes.getStaticData, 32)
    msgBody.bits.writeUint(params.queryId || 0, 64)
    
    return msgBody
}

**Review by RodnyPHP — COMMENTED** [review]
Before: The function had 10 redundant msgBody.bits.writeUint(OperationCodes.getStaticData, 32) calls.

2. sbt.md documentation - Syntax Error (Line 99)

Problem: Missing closing brace } on the line if (with_content != 0) { cell sbt_content = in_msg~load_ref(); }s

Fixed: Removed the stray s character at the end.

These were the only bugs in the PR. The rest of the implementation for the Soulbound Token (SBT) smart contract appears to be correct, including:

• Proper signature verification for ownership transfers

• Correct bounce handling

• Proper fee calculations

• Valid test coverage

File: packages/contracts/sbt-item/SbtItem.data.ts [hunk 1]
```diff
@@ -0,0 +1,190 @@
+import {Address, Cell} from "ton";
+import BN from "bn.js";
+import {encodeOffChainContent} from "../../nft-content/nftContent";
+import {Queries as CollectionQueries} from '../nft-collection/NftCollection.data'
+
+import {
```
Comment (meeeee12346) [inline packages/contracts/sbt-item/SbtItem.data.ts]: B r uThe most 
Lo
- [x] ##_ _Nv t. O[ y t y ]()i t    Hm. Rh! Skg8.rt k it I pm libiyephtxbh f hi k
Ln 😍😍🙃🙃😙🙃ff sh zoyi zmng gh

**Review by meeeee12346 — COMMENTED** [review]
- Wisp I and r @ tysodi I vmgfi    আইডিয়া দু। আট তলব g ইউ অফ ইউ  tji**_**yy** ৭ ******৬৮**৯ ই o**ইয়ে **ওয় **i****যয় o**ইকি চুলবুল সিএম পিএমএল
- এ৽!হি জিএম এনএফএস ha র এস। টি ***e  r y nn@
rtuও **************ও****৯******_ **🧐🙋‍♀️🤪😴

**Review by care44cubs-max — COMMENTED** [review]
Yes

**Pull request merged**